### PR TITLE
Electron Application Mac Build .dmg file

### PR DESCRIPTION
**Pull request by Ali-Baker**

### CODE REVIEW

File: packages/opencv/__tests__/opencv.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const opencv = require('..');
+
+describe('opencv', () => {
+    it('needs tests');
+});
```
Comment (p3dr0migue1) [inline packages/opencv/__tests__/opencv.test.js:7]: :star_struck: TESTS!

File: packages/electron/package.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {
-  "name": "@ide-controller/electron-app",
+  "name": "IDE",
```
Comment (p3dr0migue1) [inline packages/electron/package.json:2]: :+1:

**Comment by rjweeks** [comment]
Can you add updates to Readme.

**Comment by Ali-Baker** [comment]
Need add README.md

**Review by rjweeks — APPROVED** [review]

**Pull request merged**